python/tvm/relax/frontend/onnx/onnx_frontend.py

```diff
@@ -2059,12 +2059,10 @@ def _sanitize_name(self, name: str) -> str:
         if name == "":
             return self._name_supply.fresh_name("empty_")
 
-        new_name = name.replace(".", "_")
```

Comment:
it would be good to have a test case to cover this change. If this change is going to align with the description, one option is to refine the description. It would be good to avoid modifications to src/ir/supply.cc, given that it serves as the foundation for numerous cases.